os/board/rtl8720e/src/component/os/tizenrt/rtk_ble_utils.h

```diff
@@ -0,0 +1,74 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#pragma once
+
+#include <tinyara/config.h>
+#include <tinyara/net/if/ble.h>
+
+#ifdef CONFIG_AMEBAD_BLE_SCATTERNET
+extern trble_result_e rtw_ble_combo_init(trble_client_init_config* init_client, trble_server_init_config* init_server);
+extern trble_result_e rtw_ble_combo_deinit(void);
+#elif defined(CONFIG_AMEBAD_BLE_CENTRAL)
+extern trble_result_e rtw_ble_client_init(trble_client_init_config* init_parm);
+extern trble_result_e rtw_ble_client_deinit(void);
+#elif defined(CONFIG_AMEBAD_BLE_PERIPHERAL)
+extern trble_result_e rtw_ble_server_init(trble_server_init_config* init_parm);
+extern trble_result_e rtw_ble_server_deinit(void);
+#endif
```

Comment:
All the configs are AMEBAD instaed of AMEBADLITE. could you please check if this is right?

Reply:
I will change this to CONFIG_AMEBALITE